runtime/browser/ui/desktop/exclusive_access_bubble.h

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) 2012 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef XWALK_RUNTIME_BROWSER_UI_DESKTOP_EXCLUSIVE_ACCESS_BUBBLE_H_
+#define XWALK_RUNTIME_BROWSER_UI_DESKTOP_EXCLUSIVE_ACCESS_BUBBLE_H_
+
+#include "base/timer/timer.h"
+#include "ui/gfx/animation/animation_delegate.h"
+#include "ui/gfx/geometry/point.h"
+#include "url/gurl.h"
+
+namespace gfx {
+class Rect;
+}
+
+// Describes the contents of the fullscreen exit bubble.
+enum ExclusiveAccessBubbleType {
```

Comment:
I'd move it inside `ExclusiveAccessBubble` class, then it could be:
```
enum Type {
TYPE_NONE = 0,
TYPE_BROWSER_FULLSCREEN_EXIT_INSTRUCTION,
TYPE_APPLICATION_FULLSCREEN_CLOSE_INSTRUCTION
};
```